docs/setup_guide.md

```diff
@@ -0,0 +1,120 @@
+# Setup Guide
+
+This guide will provide the steps how to setup a DAO module with a Safe on Rinkeby. The setup will use Realitio and can be used with Snapshot spaces.
```

Comment:
~This guide will provide the steps how to setup a DAO module with a Safe on Rinkeby.~
This guide shows how to setup the DAO module with a Gnosis Safe on the Rinkeby testnetwork.

Add link to https://realit.io/ and to https://snapshot.org/?